modules/core_project_factory/main.tf

```
@@ -134,6 +134,15 @@ resource "google_project_service" "project_services" {
   depends_on = ["google_project.main"]
 }
 
+resource "google_project_services" "project_services_authority" {
+  count = "${ var.apis_authority == "true" ? 1 : 0 }"
```

This expression can be simplified:

```suggestion
  count = "${var.apis_authority ? 1 : 0}"
```

---

Thank you for your review. I will keep this in mind.